africanus/util/requirements.py

```diff
@@ -15,6 +15,9 @@ def _missing_packages(fn, packages, import_errors):
         return ("%s requires installation of "
                 "the following packages: %s.\n"
                 "%s" % (fn, packages, import_err_str))
+    else:
+        return ("%s requires installation of the following packages: %s. "
+                % (fn, *packages))
```

Comment:
> Ah yes, of course. I think it should pass now but note that the function now returns a list of packages instead of a tuple. Will that be problem?

Do you mean passes a list instead of a tuple?.

I think this should work:

```python
     return ("%s requires installation of the following packages: %s. "
                % (fn, tuple(packages)))
```

Reply:
Yep. I just changed it to unpack the list and modified the string it compares to accordingly.